### PR TITLE
Add total section for command line sum

### PR DESCRIPTION
**Pull request by csordasmarton**
Extend output of the `cmd sum` to contain total number of reports.

### CODE REVIEW

**Comment by csordasmarton** [comment]
Table output (`-o table`) of this feature:
```bash
------------------------------------------------------------------------------------------------------------------
Checker                | Severity | All reports | Resolved | Unreviewed | Confirmed | False positive | Intentional
------------------------------------------------------------------------------------------------------------------
misc-sizeof-expression | HIGH     | 1           | 0        | 1          | 0         | 0              | 0          
------------------------------------------------------------------------------------------------------------------
Total                  | -        | 1           | 0        | 1          | 0         | 0              | 0          
------------------------------------------------------------------------------------------------------------------
----------------------
Severity | All reports
----------------------
HIGH     | 1          
----------------------
Total    | 1          
----------------------
````

**Comment by whisperity** [comment]
Should we introduce a total row on the Web GUI too (with possibly a different row background and font size?) Or does it already contain such,  @csordasmarton?

File: libcodechecker/cmd/cmd_line_client.py [hunk 1]
```diff
@@ -647,10 +650,19 @@ def checker_count(checker_dict, key):
...
+        total['total_unreviewed'] += checker_count(unrev_checkers, key)
+        total['total_confirmed'] += checker_count(confirmed_checkers, key)
+        total['total_false_positive'] += checker_count(false_checkers, key)
+        total['total_intentional'] += checker_count(intentional_checkers, key)
 
     if args.output_format == 'json':
         print(CmdLineOutputEncoder().encode(all_results))
+        print(CmdLineOutputEncoder().encode(total))
         print(CmdLineOutputEncoder().encode(severities))
+        print(CmdLineOutputEncoder().encode([{
```
Comment (gyorb) [inline libcodechecker/cmd/cmd_line_client.py]: I do not think we should print out the total and number of all reports here.
With the current setup we print out not just one json. This api is used for further scripts to process the results, they can count it if they need it.

I think this line should be removed too:
https://github.com/Ericsson/codechecker/blob/02b70fe244c9d2dec632e8a797203b5e44a96ac6/libcodechecker/cmd/cmd_line_client.py#L663

**Comment by csordasmarton** [comment]
@whisperity I didn't find any solution to create a summary footer row with dojo DataGrid.

**Pull request merged**